src/components/App.js

```diff
@@ -1,11 +1,37 @@
+import React from 'react';
 import DisplayCalculator from './DisplayCalculator';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
-const App = () => (
-  <>
-    <DisplayCalculator />
-    <ButtonPanel />
-  </>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick = (buttonName) => {
+    const result = calculate(this.state, buttonName);
+    this.setState(result);
+  }
```

Comment:
```suggestion
  }

  handleClick = (buttonName) => {
    const result = calculate(this.state, buttonName);
    this.setState(result);
  }
```
:bulb:  When you use the arrow function syntax in JS classes, you don't need to use bind in your constructors.